crates/bevy_asset/src/loader.rs

```diff
@@ -20,22 +20,22 @@ use std::{
 };
 use thiserror::Error;
 
+pub type Error = Box<dyn std::error::Error + Send + Sync>;
```

Comment:
Maybe something like `BoxedError` to avoid confusion?

Reply:
I agree, that's better (similar to BoxedSystem?).

Reply:
renamed